### PR TITLE
created aggregation.log grouper script

### PR DESCRIPTION
**Pull request by JacksonRaffety**

### CODE REVIEW

File: aggregator-group.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    return log_list[1:]
+
+# Problems: Can't get Organization/Clustering ID for some errors
+def group_consumed_logs(logs):
+    groupings = []
+    processing_group = False
+    current_offset = 0
+    current_group = None
+
+    for i in range(len(logs)):
```
Comment (tisnik) [inline aggregator-group.py:30]: probably `for record in logs:` will make the rest of code more readable

**Review by tisnik — APPROVED** [review]
LGTM in overall, thanks a lot @JacksonRaffety

File: aggregator-group.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+
+class ConsumedGrouping:
+
+    def __init__(self):
+        self.messages = []
+        self.organization = None
+        self.cluster_id = None
+        self.error = False
+
```
Comment (svacina) [inline aggregator-group.py:13]: put class to a file, it can be shared with the pipeline script

**Review by svacina — APPROVED** [review]
more improvements after the PoC

**Pull request merged**